wiki/Forums/id.md

```diff
@@ -0,0 +1,76 @@
+# Forum
+
+![Forum osu!](main.jpg "Gambar dari forum osu!")
+
+Forum osu! merupakan salah satu fitur utama dari situs web osu!.
+Di sini, Anda dapat membuat topik atau kiriman baru dan berdiskusi bersama dalam komunitas osu!.
+
+## Subforum
+
+- [osu!](https://osu.ppy.sh/community/forums/1): _Bahasan seputar osu!_
+  - [Development](https://osu.ppy.sh/community/forums/2): _Ayo bawa osu! selangkah lebih maju. Diskusikan proyek, ide-ide baru, periklanan dan berita, rapat dan sebagainya._
+    - [Completed Projects](https://osu.ppy.sh/community/forums/116): _Proyek yang telah selesai dapat diarsipkan dan didiskusikan di sini._
```

Comment:
> Proyek yang telah selesai diarsipkan dan dapat didiskusikan di sini.

Making sure archiving a thread is only for mods right ;/ , so it should be better moving `dapat` there